src/vsl/parser/grammar/class.ne

```diff
@@ -38,11 +38,11 @@ ExtensionList
    -> delimited[type {% id %}, _ "," _] {% id %}
 
 ClassItems[s]
-   -> CodeBlock[ClassItem[$s] {% id %}] {% id %}
+   -> CodeBlock[ClassItem[$s {% id %}] {% id %}] {% id %}
 ClassItem[s]
-   -> FunctionStatement[$s] {% id %}
+   -> FunctionStatement[$s {% id %}] {% id %}
 
 InterfaceItems[s]
-   -> CodeBlock[(InterfaceItem | ClassItem[$s]) {% mid %}] {% id %}
+   -> CodeBlock[InterfaceItem {% id %}] {% id %}
```

Comment:
Interfaces can have full-functions as "default implementations". Example:

```swift
public interface Comparable<T>: Comparable<T>, Equatable<T> {
    public static func >=(lhs: Self, rhs: Self) { default implementation }
    public static func <=(lhs: Self, rhs: Self) { default implementation }
}
```
That said, interfaces can't have fields so I think we're better off as definiting clas as `ClassItem | Field` and Interfaces as `ClassItem` as interfaces can even have getters/setters:

```swift
public interface StringConvertable {
    public var asString: String { get }
}
```